c++/account_server.cc

```diff
@@ -25,12 +25,12 @@
 #include <grpcpp/grpcpp.h>
```

Comment:
Rename the folder back to `cpp`? Or is `c++` a better name?

Reply:
"cpp" has special meaning ( a submmodule) in the BUILD file for some reason, so a directory named "cpp" didn't work.  that is reason for the directory move

Reply:
How about moving workspace and build files into the `cpp` directory?

Reply:
Or, actually, keep workspace at the top level, keep only the proto targets in the top level BUILD, and add a BUILD in `cpp` for the client/server? Does that work?

Reply:
i had an issue with doing that as well!  because the build file won't let me go up directory to get to the proto directory or target and i didn't want to make copies.

Reply:
got it!  top level BUILD for proto and a separate BUILD for directory cpp (I also moved back to the originally cpp directory)